### ISSUE TITLE
Elevation Profile: Better name

### ISSUE THREAD

**Issue by Jakob-Lanstorp** [qgis/QGIS#51031]
### Feature description

It would be closer to the real world - and with the capabilities of the 'Elevation Profile' windows - to call it 'Topological Profile' or even better just 'Profile'. When you go subsurface you have several raster layers in you model - and not just an elevation. Also the fine addition of profiling vectors could be anything else than and elevation. Currently I use vector contours with water level in different groundwater reservoirs.

**Comment by Simon-Lopez** [comment]
I completely agree. "(vertical) cross-section" would make a lot more sense to me.

**Comment by Jakob-Lanstorp** [comment]
Also the QGIS issue tag label:"Profile tool" is kind of misleading for the Elevation Profile, since there also is a plugin with that name. But in reverse the word 'Profile' is a better name than 'Elevation' for the current QGIS Elevation Profile name.

**Comment by frspp** [comment]
IMO "Height profile" would be something common human understands. "Elevation profile" is still understandable. Suggestion made here are imo too jargon/too fine english for non-native speakers.

**Comment by Simon-Lopez** [comment]
> IMO "Height profile" would be something common human understands. "Elevation profile" is still understandable. Suggestion made here are imo too jargon.

According to Oxford Languages (Google):
- *cross section*: a surface or shape exposed by making a straight cut through something, especially at right angles to an axis
- *slice*: a thin, broad piece of food, such as bread, meat, or cake, cut from a larger portion
- *profile*: an outline of something, especially a person's face, as seen from one side (synonym for "a vertical cross section of a structure")

My preference goes to *cross section* because it conveys the idea of cutting into what we are observing from above.
I have no idea if these terms are too jargon. All seem to be plain English words.

**Comment by Jakob-Lanstorp** [comment]
👍  for calling it a Cross Section. If we ever gets a horizontal cross section, we could call that Slicer, so we don't need to called the cross section for vertical cross section. For the Cross Section I have 70 geological layers, that's not a Elevation Profile but a Cross Section. Any way the elevation profile in QGIS 4.0 is so cool I could live with it called Elevation Profile :)

**Comment by Simon-Lopez** [comment]
>  Any way the elevation profile in QGIS 4.0 is so cool I could live with it called Elevation Profile :)

+1 😉

**Comment by Simon-Lopez** [comment]
We really love "elevation profiles" too and are using them too to make (geological) cross-sections (https://brgm.gitlab.io/geomodelling/forgeo/)
